app/assets/stylesheets/components/sms-message.scss

```
@@ -63,7 +63,7 @@ $tail-angle: 20deg;
 .sms-message-status {
   @include core-16;
   color: $secondary-text-colour;
-  margin: -20px $gutter-half 20px $gutter-half;
+  margin: -20px govuk-spacing(3) 20px govuk-spacing(3);
```

`20px` is equivalent to `$gutter-two-thirds`
```suggestion
  margin: -govuk-spacing(4) govuk-spacing(3) govuk-spacing(4) govuk-spacing(3);
```